src/pages/backstage-comparison/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { SEO, SitewideHeader, SitewideFooter, Headline, Link } from 'components';
+import { SectionHeader, FeatureComparisonTable } from 'components/BackstageComparison';
+import { FAQs } from 'components/landing';
+import RoadieBackstageLogos from '../../../content/assets/roadie-backstage-logos.svg';
+
+const Pricing = ({
+  data: {
+    site: {
+      siteMetadata: { title: siteTitle },
+    },
+  },
+}) => {
+  return (
+    <>
+      <SEO
+        title={`Roadie vs Backstage | ${siteTitle}`}
+        description="Hosted Backstage from Roadie. Simple, flexible options to fit your IDP needs. Free Trial."
+      />
+
+      <SitewideHeader borderBottom={false} />
+
+      <div className="bg-white">
+        <div className="max-w-7xl mx-auto py-24 px-4 sm:px-6 lg:px-8">
+          <div className="sm:flex sm:flex-col sm:align-center">
+            <SectionHeader />
+          </div>
+        </div>
+        <div className="flex justify-center pb-12">
+          <img src={RoadieBackstageLogos} alt="" className="webkit-optimize-image-rendering" />
+        </div>
+        <div className="flex justify-center">
+          <div className="flex-column text-center max-w-2xl mt-5 mx-auto sm:px-6 lg:px-8">
+            <Headline el="h2" size="small">
+              Overview
+            </Headline>
+            <h2 className="text-lg sm:mt-8 lg:text-xl xl:text-xl">
+              <Link to="https://backstage.io/">Backstage.io</Link> is the best tool on the market
```

Comment:
Anyone is here probably already knows what Backstage is, we can drop the `.io` in the text.

Adding `color="primary"` makes the link easier to spot.

```suggestion
              <Link to="https://backstage.io/" color="primary">Backstage</Link> is the best tool on the market
```